site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -157,7 +159,8 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
             <TextField
               {...getFieldHelpers("name")}
               disabled={creatingWorkspace}
-              onChange={onChangeTrimmed(form)}
+              // resetMutation facilitates the clearing of validation errors
+              onChange={onChangeTrimmed(form, resetMutation)}
```

Comment:
We should not have to reset the mutation to clear the form error. But I think it is probably out of scope since it looks to be an issue with how we use the getFormHelpers gets the errors.